lms/djangoapps/teams/tests/test_views.py

```diff
@@ -122,7 +122,7 @@ def setUpClass(cls):
                     'id': 'topic_{}'.format(i),
                     'name': name,
                     'description': 'Description for topic {}.'.format(i)
-                } for i, name in enumerate([u'sólar power', 'Wind Power', 'Nuclear Power', 'Coal Power'])
+                } for i, name in enumerate([u'Sólar power', 'Wind Power', 'Nuclear Power', 'Coal Power'])
```

Comment:
I can't test the case insensitive searching because sqlite3 doesn't support it (I need the tests to pass with sqlite3).

Before, we explicitly enforced case insensitive sorting (converting to lower case), but that was expensive, and Dave O. and I decided to just rely on the MySQL behavior (which is insensitive).